juicy-draggable.html

```diff
@@ -252,7 +252,13 @@
             var element = null;
             var handle = null;
             var that = this;
-            var root = this.createShadowRoot();
+            var root;
+            if (this.attachShadow) {
+                root = this.attachShadow({ mode: 'open' });
+            } else {
+                root = this.createShadowRoot();
```

Comment:
That was actually the think I wasn't sure about. Thx, I'll fix this in other cases too.